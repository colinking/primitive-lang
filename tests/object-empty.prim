// SYNTAX TEST "source.prim" "empty objects"

namespace airplane

object TestObject123 {}
