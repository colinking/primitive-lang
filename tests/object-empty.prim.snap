>// SYNTAX TEST "source.prim" "empty objects"
#^^ source.prim comment.line.double-slash.prim punctuation.definition.comment.prim
#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.prim comment.line.double-slash.prim
>
>namespace airplane
#^^^^^^^^^ source.prim keyword.other.namespace.prim
#         ^ source.prim
#          ^^^^^^^^ source.prim entity.name.namespace.prim
>
>object TestObject123 {}
#^^^^^^ source.prim keyword.other.object.prim
#      ^ source.prim meta.object.prim
#       ^^^^^^^^^^^^^ source.prim meta.object.prim entity.name.type.prim
#                    ^ source.prim meta.object.prim
#                     ^ source.prim meta.object.prim punctuation.definition.block.prim
#                      ^ source.prim meta.object.prim punctuation.definition.block.prim
>